MDANSE_GUI/Src/MDANSE_GUI/Tabs/Plotters/Plotter.py

```diff
@@ -314,10 +320,18 @@ def plot_blank(self, *, draw_cross: bool = True):
         if draw_cross:
             axes.axline([0, 0], [1, 1], color="k", linestyle="-")
             axes.axline([0, 1], [1, 0], color="k", linestyle="-")
-        axes.set_title("The data sets you selected could not be plotted.")
-        axes.set_xlabel(
-            "If you expected a plot, please check the settings you changed last."
+        label_text = (
+            override_label
+            if override_label
+            else "If you expected a plot, please check the settings you changed last."
+        )
```

Comment:
Can be:
```suggestion
        label_text = override_label or "If you expected a plot, please check the settings you changed last."
```
if clearer.